core/pubnub_url_encode.h

```diff
@@ -4,8 +4,8 @@
 
 /* RFC 3986 Unreserved characters plus few
  * safe reserved ones. */
-#define OK_SPAN_CHARACTERS "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_.~,=;@[]"
-#define OK_SPAN_CHARS_MINUS_COMMA "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_.~=;@[]"
+#define OK_SPAN_CHARACTERS "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789-_.~[]"
```

Comment:
@650elx can you return `,` for this one back? I think tests should start passing after this change.

Reply:
@parfeon Thing is that we encode `,` as `%2C` in all of our SDKs (Python and PHP at least).
I wanted to achieve consistent behaviour across all of our SDKs :)

Reply:
@650elx when we are sending multiple channels, we send comma delimited channels. In this case, we should not encode  comma char. This is applicable to C# SDK as well.